app/components/CardListV2.tsx

```diff
@@ -0,0 +1,176 @@
+// --- React Methods
```

Comment:
Should we integrate the new platforms into the existing CardList or modify it so that we can release the updated providers individually?

Reply:
CardListV2 will be the one to work with going forward and we can remove the old cardList